src/de.hpi.swa.graal.squeak/src/de/hpi/swa/graal/squeak/model/ArrayObject.java

```diff
@@ -325,7 +328,9 @@ public static Object toObjectFromNativeObject(final NativeObject value) {
     }
 
     public void transitionFromBooleansToObjects() {
-        LOG.finer("transition from Booleans to Objects");
+        if (isLoggingEnabled) {
```

Comment:
Why is `isLoggingEnabled` needed here? Isn't there a similar check in `Log.finer()`?